codalab/apps/teams/forms.py

```diff
@@ -178,7 +178,7 @@ def __init__(self, *args, **kwargs):
     def clean_csv_file(self):
         teams_dict = {}
         if self.cleaned_data.get('csv_file'):
-            csv_team_list = [line.rstrip() for line in self.cleaned_data.get('csv_file')]
+            csv_team_list = [line.rstrip().decode('utf-8') for line in self.cleaned_data.get('csv_file')]
```

Comment:
did you test downloading these CSV's and such? we should probably have some kind of spreadsheet/matrix of features and a checklist confirming they were hit

Reply:
This one is for Organizer Teams. I tested it today and it works. For final testing on a staging server that probably makes sense.